pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -119,9 +137,11 @@ type OFBridge struct {
 
 func (b *OFBridge) CreateTable(id, next TableIDType, missAction MissActionType) Table {
 	t := newOFTable(id, next, missAction)
+
 	b.Lock()
+	defer b.Unlock()
```

Comment:
If this lock is only used to guarantee tableCache, what about using sync.Map instead of map?

Reply:
Thanks for the suggestion! Right, I am adding this to guarantee tableCache read and write. For this cache, we also have two places which iterate the values. If we change it to sync.Map, we may invoke Range method of sync.Map which needs to do type conversion for each entry and seems not widely used.